binder/environment.yml

```diff
@@ -2,34 +2,36 @@ channels:
 - conda-forge
 dependencies:
 # Jupyter
-- jupyterlab=2
-- nodejs=11
-- nbconvert=5.5
-- widgetsnbextension=3.5
+- jupyterlab=3
+- jupyter-offlinenotebook=0.2
+- nodejs=12
+- nbconvert=6
 # Python Kernel
 - ipykernel=5.1
-- ipywidgets=7.5
-- ipyleaflet=0.12
-- altair=3.1
-- bqplot=0.12
-- dask=2.1
+- xeus-python=0.9
+- ipywidgets=7.6
+- widgetsnbextension=3.5
+- ipyleaflet=0.13
+- altair=4.1
+# - bqplot=0.12
+- dask=2020.12
 - matplotlib=3.1
-- pandas=0.24
+- pandas=1
 - python=3.7
 - scikit-image=0.15
 - scikit-learn=0.21
-- seaborn=0.9
+- seaborn=0.11
 - tensorflow=1.13
 - sympy=1.4
 - pyyaml
 - traittypes==0.2.1
 - invoke=1.2
 # C++ Kernel
-- xeus-cling=0.6
+- xeus-cling=0.10
```

Comment:
To answer your question @SylvainCorlay, this syntax installs `0.10.1`:

![image](https://user-images.githubusercontent.com/591645/103900397-0a4f1b00-50f8-11eb-8890-0cf2bc267a77.png)